javascript/ql/lib/semmle/javascript/frameworks/SQL.qll

```diff
@@ -174,6 +179,16 @@ private module Postgres {
   private class QueryCall extends DatabaseAccess, DataFlow::MethodCallNode {
     QueryCall() { this = [client(), pool()].getMember("query").getACall() }
 
+    override DataFlow::Node getAResult() {
+      this.(DataFlow::CallNode).getNumArgument() = 2 and
+      result = this.(DataFlow::CallNode).getCallback(1).getParameter(1)
+      or
+        this.(DataFlow::CallNode).getNumArgument() = 1 and
+        result = this.getAMethodCall("then").getCallback(0).getParameter(0)
+      or
+        PromiseFlow::loadStep(this.getALocalUse(), result, Promises::valueProp())
+    }
+
```

Comment:
I think the autoformatter will fail on this.

Reply:
I had no idea the autoformatter existed 🙈